### DISCUSSION TITLE
macOS 下 Python3 提示 No module named _tkinter — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#275] [Announcements]
# macOS 下 Python3 提示 No module named _tkinter — 码志

在 macOS 下运行一个依赖 tkinter 的 Python3 脚本时，提示：

https://mazhuang.org/fragment/macos-python3-tkinter/

**Comment by wastu01** [comment]
謝謝此篇文章，解決我遇到的問題

**Comment by radarwang** [comment]
ddd
